contracts/TokenTypeInterface.sol

```diff
@@ -0,0 +1,23 @@
+/*
+ * Hubii Striim
+ *
+ * Compliant with the Hubii Striim specification v0.12.
+ *
+ * Copyright (C) 2017-2018 Hubii AS
+ */
+
+pragma solidity ^0.4.24;
+
+/**
+@title TokenTypeInterface
+@notice A base contract to handle different token types
+*/
+contract TokenTypeInterface {
+    event TokenTransferred(address token, address from, address to, uint256 amount, uint256 id);
+
+    function isTyped() public view returns(bool);
+    function isQuantifiable() public view returns(bool);
```

Comment:
I don't quite see why we need the pair of functions `isTyped()` and `isQuantifiable()`. If I interpret your function names correctly they both describe aspects or consequences of fungibility. I.e. a fungible asset will be quantifiable but will not be typed. A non-fungible one will not be quantifiable but will indeed be typed. Could we rather just have _one_ function of signature `function isFungible() public view returns(bool)`?

Reply:
I know now is useless but might be in a future we need to handle a contract with both.